jax/experimental/jax2tf/tests/shape_poly_test.py

```diff
@@ -1276,18 +1276,55 @@ def _make_harness(group_name: str, name: str,
                   jax.grad(lambda x: jnp.sum(jnp.concatenate([x, x], axis=0))),
                   [RandArg((3, 4, 5), _f32)],
                   poly_axes=[(0, 1)]),
+
+    # Issue #11402 InconclusiveDimensionOperation: Dimension polynomial '-1*t' is not a multiple of '2'
```

Comment:
This will be fixed in an upcoming PR. This fails with enable_xla=False and True, so it is a high priority.